src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -157,17 +157,38 @@ public sealed class ClusterSingletonOptions
     public sealed class ShardOptions
     {
         /// <summary>
+        ///     <para>
         ///     Defines how the coordinator stores its state. The same setting is also used by the
-        ///     shards for RememberEntities.
+        ///     shards when <see cref="RememberEntities"/> is set to <c>true</c>.
+        ///     </para>
+        ///     
+        ///     Possible values are <see cref="Akka.Cluster.Sharding.StateStoreMode.Persistence"/> and
+        ///     <see cref="Akka.Cluster.Sharding.StateStoreMode.DData"/>
+        /// </summary>
+        public StateStoreMode? StateStoreMode { get; set; }
+
+        /// <summary>
+        ///     <para>
+        ///     When <see cref="RememberEntities"/> is enabled and the state store mode is
+        ///     <see cref="Akka.Cluster.Sharding.StateStoreMode.DData"/>, this controls how the remembered entities
+        ///     and shards are stored.
+        ///     </para>
+        ///
+        ///     <para>
+        ///     Possible values are <see cref="Akka.Cluster.Sharding.RememberEntitiesStore.Eventsourced"/> and
+        ///     <see cref="Akka.Cluster.Sharding.RememberEntitiesStore.DData"/>
+        ///     </para> 
+        /// 
+        ///     Default is <see cref="Akka.Cluster.Sharding.RememberEntitiesStore.DData"/> for backwards compatibility.
```

Comment:
I think the default here should be none / ignored by default: https://getakka.net/community/whats-new/akkadotnet-v1.5-upgrade-advisories.html - so I'd set this to `null` I suppose and handle that accordingly.

Reply:
I think your code looks ok - it might just be the XML-DOC comment that's wrong.

Reply:
https://github.com/akkadotnet/akka.net/blob/c9ccc25207b5a4cfa963a5a23f96c0676fbbef10/src/contrib/cluster/Akka.Cluster.Sharding/reference.conf#L26-L29

Reply:
Sigh, it's wrong in both places then. Let's fix it here and then submit a PR to do it in Akka.NET as well.

Reply:
So if the user did not set the `RememberEntitiesStore` property, the default value would be taken from the fallback default configuration, which is `"ddata"`

Reply:
Good catch 🔍